microservices/gateway/src/interfaces/remote-config.ts

```diff
@@ -5,4 +5,5 @@ import type { CorsOptions } from 'cors';
  */
 export interface IRemoteConfig {
   corsOptions?: CorsOptions;
+  webhookOptions?: { url: string; allowMethods: string[] };
```

Comment:
```suggestion
  webhookOptions?: { url?: string; allowMethods?: string[] };
```

Reply:
is it possible to leave this option? Because we always have default option for this.
![image](https://user-images.githubusercontent.com/48718981/235798010-589444b5-8621-4b77-b22b-fc313538f359.png)

Reply:
We describe remote config, not constants